Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1416,22 +1433,25 @@
     <RegEx find="anić" replaceWith="anit ć" />
     <RegEx find="(?&lt;!gl|[Nn])avić" replaceWith="avit ć" />
     <RegEx find="\bdral" replaceWith="deral" />
+    <RegEx find="dranj" replaceWith="deranj" />
     <RegEx find="dsek" replaceWith="dsjek" />
     <RegEx find="ebać" replaceWith="ebat ć" />
     <RegEx find="(?&lt;![NnJj])edać" replaceWith="edat ć" />
     <RegEx find="eizbež" replaceWith="eizbjež" />
     <RegEx find="efinišu" replaceWith="efiniraju" />
     <RegEx find="erišu" replaceWith="eriraju" />
-    <RegEx find="fikova([nlot])" replaceWith="ficira$1" />
+    <RegEx find="(mpl|st|f)ikova([nlot])" replaceWith="$1icira$2" />
     <RegEx find="fikuj" replaceWith="ficir" />
     <RegEx find="fisa" replaceWith="fira" />
     <RegEx find="fiše" replaceWith="fira" />
     <RegEx find="frov" replaceWith="frir" />
     <RegEx find="(?&lt;!\b[Oo]zlo|\b[Ii]sp(rip)?ov|i)jeđen" replaceWith="ijeđen" />
     <RegEx find="(?&lt;!hva)lisati" replaceWith="lirati" />
     <RegEx find="jći" replaceWith="jući" />
+    <RegEx find="([kKPp])ratić" replaceWith="$1ratit ć" />
     <RegEx find="koriš[čć]a" replaceWith="korišta" />
-    <RegEx find="lizovan" replaceWith="liziran" />
+    <RegEx find="([nl])izovan" replaceWith="$1iziran" />
+    <RegEx find="(?&lt;![Oo]b)([nl])izuje" replaceWith="$2izira" />
```

Comment:
```xml
    <RegEx find="(?&lt;![Oo]b)([nl])izuje" replaceWith="$1izira" />
```
The first group is zero-width negative look-behind (i.e., a non-capturing group).